### ISSUE TITLE
PS3 Physical Remote random repeats direction often in XMB

### ISSUE THREAD

**Issue by jt-gilkeson** [aldostools/webMAN-MOD#1094]
If you are using the official PS3 Remote Control to navigate the XMB when webMAN-MOD is installed, it randomly (very often) repeats navigation in the last direction pushed which makes it very annoying to try to get around the XMB.  Unloading webMAN (via menu) "fixes" it.  But the remote control should work normally in the XMB without having to disable webMAN - like if  I'm watching a movie or using UMS to stream something.

**Comment by aldostools** [comment]
Verify if you're using the tool in game mode.

PS3 Remote Control has 2 modes: Normal and Game modes. You can switch between them using the top right menu or clicking on the label under the Remote IP.

Normal mode is for use in XMB, Movían, backup managers, etc.

Game mode keeps the last button pressed until the next button is selected. It's mainly intended for some games.

**Comment by jt-gilkeson** [comment]
I'm talking about the official physical (bluetooth)  "Sony PlayStation 3 Blu-ray Disc Remote Control" not the tool.  This thing: https://www.amazon.com/Sony-PlayStation-3-Blu-ray-Disc-Remote/dp/B000M17AVO?th=1

**Comment by aldostools** [comment]
Oh I thought you were talking about my PS3 Remote Control http://ps3.aldostools.org

I don't know why that device is having conflicts with my tool. webMAN MOD use http requests for web commands.
The remote control uses bluetooth. Maybe the buttons are sticky or the device is defective.

**Comment by jt-gilkeson** [comment]
Nope, it's not a physical issue.  Unloading webman mod never repeats navigation (100% of the time), yet if I have webman mod on, it happens (repeats navigation at random - but often).  I encountered it right away when I put CFW on my system and installed webman mod. I never had the issue on stock FW  It wasn't until recently that I confirmed that turning off webman mod "fixes" the issue.  So it appears it's either something in webman MOD or some issue that gets exposed when webman MOD is loaded in memory?  

https://www.psx-place.com/threads/cfw-4-91-2-evilnat-cobra-8-5-beta9-cex-dex-pex-d-pex.39743/page-24#post-364937

If there are any logs that would help or things I could turn on to debug, please let me know.

**Comment by aldostools** [comment]
You could enable Cobra debug in Network > Custom Firmware Tools > Cobra menu of Evilnat and use [socat](https://sourceforge.net/projects/unix-utils/files/socat/1.7.3.2/) to check the logs.
```
socat -u udp-recv:18194 stdout
```

I find it a really rare issue since webMAN MOD mainly receives network commands. 
Unless some memory patch (poke) is causing the issue.

You could check if webMAN MOD Lite have a different behavior. Since it has less features than the full edition.
